lib/settings_base.py

```diff
@@ -157,6 +157,7 @@ def lazy_langs(languages):
 
 # Where product details are stored see django-mozilla-product-details
 PROD_DETAILS_DIR = path('lib', 'product_json')
+PROD_DETAILS_URL = 'https://svn.mozilla.org/libs/product-details/json/'
```

Comment:
I've seen in the bug there'll be https://product-details.mozilla.org/ soon (?), which'll be even better ;)